qfdmo/models/acteur.py

```diff
@@ -595,6 +597,56 @@ def create_parent(self):
         self.save()
         return revision_acteur_parent
 
+    def duplicate(self):
+        if self.is_parent:
+            raise Exception("Impossible de dupliquer un acteur parent")
+
+        # Get linked objects before removing pk
+        labels = self.labels.all()
+        acteur_services = self.acteur_services.all()
+        # Besoin de cette "list comprehension" pour executer la Queryset,
+        # sinon elle n'est pas executé avant que le revisionacteur soit modifié
+        proposition_services = [ps for ps in self.proposition_services.all()]
+
+        acteur = Acteur.objects.get(identifiant_unique=self.identifiant_unique)
+
+        nullable_fields = [
```

Comment:
```suggestion
        fields_to_reset = [
```

un truc comme ça ce serait pas plus clair ? 
c'est bien des champs que tu veux réinitialiser ?